config/deploy.rb

```diff
@@ -52,14 +52,25 @@
 end
 
 task :'db:configure' do
-  if File.exist?("#{deploy_to}/shared/config/dbExists")
-    queue %[echo "Existing installation, migrating"]
-    invoke :'rails:db_migrate'
-  else
-    queue %[echo "Fresh installation, using db:setup"]
-    invoke :'fresh:db'
-    queue! %[touch "#{deploy_to}/#{shared_path}/config/dbExists"]
-  end
+  queue! %{
+    targetpath="#{deploy_to}/#{shared_path}/config/dbExists"
+    if [ -e "$targetpath" ]
+    then
+      echo "Existing installation detected! Using rake db:migrate."
+      rake db:migrate
+    else
+      echo "Fresh VM installation detected! Using rake db:setup."
+      read -r -p "Blank installation detected, are you sure you wish to continue? [y/n] " response
+      if [[ $response =~ ^([yY][eE][sS]|[yY])$ ]]
+        then
+          rake db:setup
```

Comment:
Unbundle if we can swap destructive commands for passive commands -- does is `db:migrate` sufficient to replace `db:schema:load`?